### DISCUSSION TITLE
Where are "Issues"? I can't find it, I can't make any of it

### DISCUSSION THREAD

**Discussion by VadiqueMe** [community/community#22747] [New to GitHub]
<p>I have the project here <a href="https://github.com/VadiqueMe/dogecoin" rel="nofollow noopener">https://github.com/VadiqueMe/dogecoin</a> , but I can’t find how to create any issue for it_(*_, to summarize some of existing gotchas, for example. Where is it? Has it been [re]moved or what?</p>
<p><em>*)</em> for this my fork exactly, not for upstream’s variant</p>
<p><em>update</em> I see only the link to <a href="https://github.com/issues" rel="nofollow noopener">https://github.com/issues</a> on the top bar, which lists every issue I participated in, but not giving me “issues” for this exact project</p>
<p><em>update too</em> And for this project <a href="https://github.com/VadiqueMe/PhysicsOfElasticContinua" rel="nofollow noopener">https://github.com/VadiqueMe/PhysicsOfElasticContinua</a> , I have the wanted “issues” just next to “code”. I wish to get the same for <a href="https://github.com/VadiqueMe/dogecoin" rel="nofollow noopener">https://github.com/VadiqueMe/dogecoin</a></p>

**Comment by DilumAluthge** [comment, marked as answer]
<p>Try going to your repo’s settings page: <a href="https://github.com/VadiqueMe/dogecoin/settings" rel="nofollow noopener">https://github.com/VadiqueMe/dogecoin/settings</a></p>
<p>Scroll down to “Features” and check the box next to “Issues.”</p>
